pkg/sdk/poc/generator/file_util.go

```diff
@@ -1,24 +0,0 @@
-package generator
-
-import (
-	"bytes"
-	"go/format"
-	"log"
-	"os"
-	"path/filepath"
-)
-
-func WriteCodeToFile(buffer *bytes.Buffer, fileName string) {
```

Comment:
I prefer to have it in separate util, not in util.go, because there are different levels of utils there.